feat(curriculum): add chp3 md2 sharing personal details (#64546)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/blocks/es-a1-learn-basic-personal-information/69335252457f54a2405b3ea9.md

---
id: 69335252457f54a2405b3ea9
title: Task 14
challengeType: 22
dashedName: task-14
lang: es
---

<!-- (Audio) Esteban: Soy desarrollador web. -->

# --description--

> In this task, you will hear Esteban answered Ángela when asked about his profession.

Reviewer: ```suggestion
In this task, you will hear Esteban's answer to Ángela's question about his profession.
```
